RecoEgamma/Examples/plugins/PatPhotonSimpleAnalyzer.cc

```diff
@@ -17,25 +17,104 @@
 //
 //         Created:  Wed Sep 23 12:00:01 CDT 2008
 //
-///////////////////////////////////////////////////////////////////////
```

Comment:
Thanks for spotting that!

Reply:
Wait, is this not correct after all? In the original header file, we had:
```
/**\class PatPhotonSimpleAnalyzer
 Description: Analyzer to make a load of histograms for the improvement of the PhotonID object
 Implementation:
     \\\author: J. Stilley, A. Askew May 2008
*/
//
```

In the original `.cc` file, we had:

```
// Original Author:  M.B. Anderson
//   based on simple photon analyzer by:  J. Stilley, A. Askew
```

I just kept the mention in the `.cc` file, because it says `based on simple photon analyzer by:  J. Stilley, A. Askew`. For me, this hinted that the `\\\author: J. Stilley, A. Askew May 2008` was just a copy-and-paste from this simple photon analyzer that the plugin was based on. Hence ```// Original Author:  M.B. Anderson, based on simple photon analyzer by:  J. Stilley, A. Askew``` made sense for me to keep.